src/main.rs

```diff
@@ -53,6 +53,8 @@ async fn main() -> Result<()> {
         .init();
 
     let mut conversation = Conversation::new(args.goal);
+    std::fs::create_dir_all("./browser")?;
+    std::fs::create_dir_all("./user_data")?;
```

Comment:
I'd rather we move this into the `browser::init` call, since the docs already state it'll create the paths if they don't exist.